resources/todomvc/big-dom-generator/utils/buildComplex.js

```diff
@@ -0,0 +1,76 @@
+const fs = require("fs").promises;
+const { JSDOM } = require("jsdom");
+const path = require("path");
+
+const TARGET_DIRECTORY = "./dist";
+const COMPLEX_DOM_HTML_FILE = "index.html";
+const TODO_HTML_FILE = "index.html";
+const CSS_FILES_TO_ADD_LINKS_FOR = ["big-dom-generator.css", "generated.css"];
+
+async function buildComplex(CALLER_DIRECTORY, SOURCE_DIRECTORY, TITLE, FILES_TO_MOVE) {
+    // remove dist directory if it exists
+    await fs.rm(path.resolve(TARGET_DIRECTORY), { recursive: true, force: true });
+
+    // re-create the directory.
+    await fs.mkdir(path.resolve(TARGET_DIRECTORY));
+
+    // copy dist folder from javascript-es6-webpack
+    await fs.cp(path.join(CALLER_DIRECTORY, SOURCE_DIRECTORY), path.resolve(TARGET_DIRECTORY), { recursive: true }, (err) => {
+        if (err)
+            console.error(err);
+    });
+
+    // copy files to move
+    for (let i = 0; i < FILES_TO_MOVE.length; i++) {
+        // rename app.css to big-dom-generator.css so it's unique.
+        const sourcePath = path.resolve(CALLER_DIRECTORY, "..", FILES_TO_MOVE[i]);
+        const fileName = path.basename(FILES_TO_MOVE[i]);
+        const targetPath = fileName === "app.css" ? path.join(TARGET_DIRECTORY, "big-dom-generator.css") : path.join(TARGET_DIRECTORY, fileName);
+        await fs.copyFile(sourcePath, targetPath);
+    }
+
+    // read todo.html file
+    console.log(path.resolve(CALLER_DIRECTORY, path.join("..", "dist", TODO_HTML_FILE)));
```

Comment:
If you want to keep this log, please add some more context to the message.
Also please put the `path.resolve` part to a temporary variable.

Reply:
Removed. Wasn't supposed to keep.